### DISCUSSION TITLE
Update emule references (servers / nodes / ipfilter)

### DISCUSSION THREAD

**Discussion by ansani** [ansani/Shareaza#43] [Ideas]
The reference to peerates.net server list is now old (no direct download is possible anymore).
I plan to use the server list provided by Emule Security (same for nodes and ipfilter).

What do you think?

**Comment by ansani** [comment]
The new server.met URI will be http://upd.emule-security.org/server.met

**Reply by Mika3578** [reply]
good the best choice
